lib/records/exporter.rb

```diff
@@ -137,7 +139,7 @@ def self.should_keep_entry(e, attribute, record, cutoff_date)
         when :careplans
           return record.careplan_active?(e['type'])
         when :conditions
-          return record.present[e['type']] || (e['end_time'] && e['end_time'] > cutoff_date)
+          return record.present[e['type']] || (e['end_time'] && e['end_time'] > cutoff_date && e['end_time'] <= end_time)
```

Comment:
Not sure this is correct...

This keeps the condition of it is active (present) or if the end_time is after the cut-off and before the end... Theoretically, it should keep the condition if it ended after the cut-off... even if it was in the future.